src/Illuminate/Auth/Notifications/ResetPassword.php

```diff
@@ -18,14 +18,14 @@ class ResetPassword extends Notification
     /**
      * The callback that should be used to create the reset password URL.
      *
-     * @var \Closure|null
+     * @var (\Closure(mixed, string): string)|null
      */
     public static $createUrlCallback;
 
     /**
      * The callback that should be used to build the mail message.
      *
-     * @var \Closure|null
+     * @var (\Closure(mixed, string): \Illuminate\Notifications\Messages\MailMessage)|null
```

Comment:
It should also be possible to return a Mailable (I have never tried that).
```suggestion
     * @var (\Closure(mixed, string): (\Illuminate\Notifications\Messages\MailMessage|\Illuminate\Mail\Mailable))|null
```

Reply:
The usage class doesn't return what you suggesting. Maybe create different PR. I'm not adding something I can't confirm right now :-)